gef.py

```diff
@@ -5559,10 +5573,19 @@ def do_invoke(self, argv):
         if self.get_setting("clear_screen"):
             clear_screen(redirect)
 
-        for pane in current_layout:
-            if pane[0] in ("!", "-"):
+        do_warn = False  # Deprecating "!"
+        for section in current_layout:
+            # Deprecating "!" from the layout syntax
+            if section[0] == "!":
+                do_warn = True
+                continue
+            if section[0] == "-":
                 continue
-            layout_mapping[pane]()
+            layout_mapping[section]()
+
+        # Deprecating "!"
+        if do_warn:
+            warn("context.layout: '!' deprecated: Use '-' before sections to hide them.")
```

Comment:
Awesome, let's leave it like this for a few. Could you add that the `!` will not work in the future release? To force ppl update their config file.